### DISCUSSION TITLE
funny medic-bot's skill description)

### DISCUSSION THREAD

**Discussion by 7952812** [FakeFishGames/Barotrauma#13975] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

during a session with 2 players (captain and mechanic) and 2 bots (engineer and medic) i found an itneresting and funny bug in the description of medic's skill: 
Medical insurance
**Grants [xpbonus]% more experience and [moneybonus]% more
money for [missiontype] missions."**
![image](https://github.com/FakeFishGames/Barotrauma/assets/170206566/3922d513-28a3-4021-8b90-6e165ff3a37f)
language is russian.

### Reproduction steps

i've just opened the medic's skill menu in Information (TAB button). 

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13839
